README.md

```diff
@@ -48,9 +48,11 @@ The permissions required are broken into 2 groups:
 * WRITE: `RecordAdd`, `RecordDelete`, `ZonePublish`, `ZoneDiscardChangeset`, `ZoneFreeze`, `ZoneThaw`, `ZoneAddNode`,
 `ZoneRemoveNode`
 
-All CI validations only require READ permissions; deplyoing requires a user with READ and WRITE permissions.
+All CI validations only require READ permissions; deploying requires a user with READ and WRITE permissions.
 
-For a breakdown of what each permssion allows read through [DynECT's permissions guide](https://help.dyn.com/user-and-group-permissions/).
+In addition, the `AliasService` permission is required to be able to read or write ALIAS records on DynECT.
```

Comment:
there were two reasons... 1. this 2. the api version used (the fog-dynect gem used an old api version that didn't support ALIAS - ie. dynect wouldn't accept OR return ALIAS records)